.storybook/webpack.config.js

```diff
@@ -34,5 +36,13 @@ module.exports = (storybookBaseConfig, configType, defaultConfig) => {
     }
   })
 
+  defaultConfig.plugins.push(new StyleLintPlugin({
+    files: [
+      'components/**/*.vue',
+      'assets/styles/**/*.scss'
+    ],
```

Comment:
we have also `scss` styles in components directory. e.g `/components/02-elements/button/Button.scss` We are going to divide all components to separate files so I'm not sure if you cover these files?

Reply:
Good catch! These files were omitted :+1: